spec/lib/ability_spec.rb

```diff
@@ -66,7 +66,7 @@
       let(:facility) { Facility.cross_facility }
 
       it { is_expected.to be_allowed_to(:manage_accounts, facility) }
-      it_is_allowed_to([:read, :edit, :suspend, :unsuspend], Account)
```

Comment:
Yup, looks like `:create` covers `:new` as well, and `:update` covers `:edit`. Updated the ability file accordingly.